### PR TITLE
Feature/frontend/views/app skeleton

### PR DESCRIPTION
**Pull request by rem1niscence**
Consist of three main components: Breadcrumbs, AppSidebar and AppNavbar, as well as minor refactors in other components in order to give them more flexibility.

### CODE REVIEW

File: frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import Sidebar from "../../Sidebar";
+import {Row} from "react-bootstrap";
+import MenuItem from "./MenuItem/MenuItem";
+import "./AppSidebar.scss";
+
+class AppSidebar extends Component {
+  render() { 
+    return <React.Fragment>
```
Comment (xerp) [inline frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js]: Why **Fragment** here? this has more sense on **MenuItem** not here.

Here, we have the **Sidebar** element as root for this component.

File: frontend/src/core/components/Dashboard/LoginStatus.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, {Component} from "react";
+import {Dropdown} from "react-bootstrap";
+import PropTypes from "prop-types";
+
+
+// eslint-disable-next-line react/display-name
+const LabelToggle = React.forwardRef(({children, onClick}, ref) => (
+  <React.Fragment>
```
Comment (xerp) [inline frontend/src/core/components/Dashboard/LoginStatus.js]: Same comment about **Fragment**

File: frontend/src/core/components/Dashboard/LoginStatus.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  children: PropTypes.any,
+  onClick: PropTypes.func
+};
+
+class LoginStatus extends Component {
+  state = {  }
+  render() { 
+    return <Dropdown>
+    <Dropdown.Toggle as={LabelToggle} id="dropdown-basic">
+      XXX_JOKER_XXX
```
Comment (xerp) [inline frontend/src/core/components/Dashboard/LoginStatus.js]: Make the **LoginStatus** takes username from a **props**.

**Review by xerp — CHANGES_REQUESTED** [review]
See the comments about CR.
Also change the Dashboard on **core/components/Dashboard**, c to **DefaultLayout**.
And Dashboard on **views/Dashboard** will be part of SPA page.

File: frontend/src/core/components/Dashboard/LoginStatus.js (hunk 2 above)
Comment (rem1niscence) [reply frontend/src/core/components/Dashboard/LoginStatus.js]: As here I'm using two sibling elements, should I replace it with a div?

**Comment by rem1niscence** [comment]
> And Dashboard on **views/Dashboard** will be part of SPA page.

Could you please elaborate on this?

**Comment by xerp** [comment]
> > And Dashboard on **views/Dashboard** will be part of SPA page.
> 
> Could you please elaborate on this?

The **Dashboard** will be the _home_ page(after login) of the app, not the _skeleton itself_. The skeleton is the **DefaultLayout** component

**Review by xerp — CHANGES_REQUESTED** [review]
The change made looks good. Only the dashboard issue remains.

File: frontend/src/_routes.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ export const routePaths = {
  */
 const pageRoutes = [
   {path: routePaths.login, exact: true, name: "Login", component: Login},
-  {path: routePaths.home, exact: true, name: "Dashboard", component: Dashboard},
+  {path: routePaths.home, exact: true, name: "Dashboard", component: DefaultLayout},
```
Comment (xerp) [inline frontend/src/_routes.js]: Change the name of this route to another one.

**Review by xerp — CHANGES_REQUESTED** [review]
Only one issue remains

File: frontend/src/core/components/Dashboard/LoginStatus.js (hunk 2 above)
Comment (xerp) [reply frontend/src/core/components/Dashboard/LoginStatus.js]: yes, according to this pattern: https://reactjs.org/docs/fragments.html

File: frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js (hunk 1 above)
Comment (xerp) [reply frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js]: @justaname94 What about this one?

File: frontend/src/core/components/Dashboard/LoginStatus.js (hunk 3 above)
Comment (xerp) [reply frontend/src/core/components/Dashboard/LoginStatus.js]: @justaname94 What about this one?

**Review by xerp — CHANGES_REQUESTED** [review]
Disregard the last comment, because there are more request changes.

File: frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js (hunk 1 above)
Comment (rem1niscence) [reply frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js]: I changed it on 08a7289

File: frontend/src/core/components/Dashboard/LoginStatus.js (hunk 3 above)
Comment (rem1niscence) [reply frontend/src/core/components/Dashboard/LoginStatus.js]: I changed it on 08a7289

**Review by xerp — COMMENTED** [review]
The latest changes look good, only remain the other issues

**Review by xerp — APPROVED** [review]
The PR looks good.

**Pull request merged**